### DISCUSSION TITLE
Switching Back from GitHub Copilot Enterprise to Business Plan

### DISCUSSION THREAD

**Discussion by jumpe1** [community/community#126137] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi everyone,

I am currently using GitHub Copilot Business and considering upgrading to GitHub Copilot Enterprise. However, I would like to know if it is possible to switch back to the Business plan after upgrading to the Enterprise plan. Has anyone gone through this process or have any insights on how this can be done? Any information on the steps involved and potential challenges would be greatly appreciated.

Thank you!

**Comment by arairyus** [comment, marked as answer]
@jumpe1 

https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-copilot/managing-your-github-copilot-enterprise-subscription#downgrading-your-copilot-enterprise-subscription

This link may help you.
